llvm/lib/Target/X86/X86.td

```diff
@@ -1825,32 +1818,32 @@ def : ProcModel<P, SapphireRapidsModel,
 
 def : Proc<"k6",   [FeatureX87, FeatureCX8, FeatureMMX],
                    [TuningSlowUAMem16, TuningInsertVZEROUPPER]>;
-def : Proc<"k6-2", [FeatureX87, FeatureCX8, Feature3DNow],
+def : Proc<"k6-2", [FeatureX87, FeatureCX8],
```

Comment:
These still need FeatureMMX and SSEPrefetch (or a similar flag)

Reply:
Oops! I had fixed that but in a not-yet-committed diff in my checkout. (Tests fail, otherwise!)